### ISSUE TITLE
Error ingesting a video

### ISSUE THREAD

**Issue by jrochkind** [sciencehistory/scihist_digicoll#2052]
In Kithe::CreateDerivativesJob

For https://digital.sciencehistory.org/admin/asset_files/ed4x3m1

We get an error:

[scihist_digicoll/production] TTY::Command::ExitError: Running `vipsthumbnail /tmp/temp_deriv20230131-37-nodbej.jpg --eprofile /app/vendor/bundle/ruby/3.1.0/gems/kithe-2.8.0/lib/vendor/icc/sRGB2014.icc --delete --size 54x65500 -o /tmp/kithe_vips_cli_image_to_jpeg20230131-37-zwx88r.jpg\[Q\=85,interlace,optimize_coding,strip\]` failed with  exit status: 255  stdout: Nothing written  stderr: vipsthumbnail: unable to thumbnail /tmp/temp_deriv20230131-37-nodbej.jpgVipsForeignLoad: "/tmp/temp_deriv20230131...

## Backtrace

    line 148 of [PROJECT_ROOT]/app/uploaders/asset_uploader.rb: block (2 levels) in <class:AssetUploader>
    line 145 of [PROJECT_ROOT]/app/uploaders/asset_uploader.rb: each_pair
    line 145 of [PROJECT_ROOT]/app/uploaders/asset_uploader.rb: block in <class:AssetUploader>

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/58989/faults/93062846)

-------

The ingested file looks like it's a good mp4 video, I can at least start to play it. 

Not sure why it's running `vipsthumbnail` on video... do we normally even do that for video, don't we only do that for images?

**Comment by jrochkind** [comment]
OK that same video failed it's HLS creation job on AWS MediaConvert:

> Error initializing encoder for video target [4] [initialization failed: Invalid level specified for the selected framerate or resolution.]

I think there's something not right with that mp4....

**Comment by jrochkind** [comment]
Cc @archivistsarah

**Comment by jrochkind** [comment]
ffprobe on the video does report some errors, although it is also able to report metadata

> [mov,mp4,m4a,3gp,3g2,mj2 @ 0x11c605720] st: 1 edit list: 1 Missing key frame while searching for timestamp: 0
> [mov,mp4,m4a,3gp,3g2,mj2 @ 0x11c605720] st: 1 edit list 1 Cannot find an index entry before timestamp: 0.

**Comment by archivistsarah** [comment]
There was something weird with the audio and video sync on this file that caused all the errors, although I can't tell you exactly what it was. I can say that there was no audio in the video after the first 30 seconds, which is an error (original has audio throughout) and it seems like this was caused by the way DVDs sync chapters, frames, and audio (as this was from a DVD, not a VHS tape).  I generated this file with a new process for mp4 creation and clearly ran into some issues! Redid the mp4 export with the old process and the file is fine. I think we can close this and chalk it up to a bad file, not anything wrong with the HLS or thumbnail process.
